docs/releasing/publishing.md

```diff
@@ -15,8 +15,9 @@ Developers should pair on releases. When remote working, it can be useful to be
 4. Create and check out a new branch (`release-[version-number]`). See the [versioning documentation](/docs/contributing/versioning.md) for more information.
 
 5. Update the [`CHANGELOG.md`](../../CHANGELOG.md) by:
-  - changing the 'Unreleased' heading to the new version-number and release-type - for example, '3.11.0 (Feature release)'
-  - adding a new 'Unreleased' heading above the new version-number and release-type, so users will know where to add PRs to the changelog
+
+- changing the 'Unreleased' heading to the new version-number and release-type - for example, '3.11.0 (Feature release)'
+- adding a new 'Unreleased' heading above the new version-number and release-type, so users will know where to add PRs to the changelog
```

Comment:
I think those are meant to be part of the 5. item:

```suggestion
   - changing the 'Unreleased' heading to the new version-number and release-type - for example, '3.11.0 (Feature release)'
   - adding a new 'Unreleased' heading above the new version-number and release-type, so users will know where to add PRs to the changelog
```

Reply:
Thanks for this, fixed the bullet indenting 👍